modules/map-canvas.js

```diff
@@ -1,6 +1,48 @@
 import {MAP_STYLES} from "./map-canvas-styles.js";
 import {ImageDataConverter} from "./image-data-converter.js";
 
+Hooks.once('ready', async function() {
```

Comment:
Can this be rolled into the registerSettings() function?

Reply:
yes. that makes sense.